### ISSUE TITLE
Make info tooltips tabbable so they can be focused and shown via keyboard navigation

### ISSUE THREAD

**Issue by techanvil** [google/site-kit-wp#11681]
## Feature Description

Keyboard users should be able to focus info tooltip icons in order to view the tooltip.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Info tooltip icons should tabbable, and the tooltip should appear when the icon is navigated to via the keyboard.

## Implementation Brief

- [ ] In `assets/js/components/InfoTooltip.js`:
  - Add the props `tabIndex="0"` and `role="tooltip"` to the rendered `span` element.

### Test Coverage

* Fix any failing tests.

## QA Brief

* 

## Changelog entry

*

**Comment by eugene-manuilov** [comment]
IB ✔️

**Comment by cjadown** [comment]
QA Update ✅

- Tested on the Develop Branch
- Verified that the Info tooltip icons are tabbable, and that the tooltip appears when the icon is navigated to via the keyboard

<Details>

<img width="2456" height="1271" alt="Image" src="https://github.com/user-attachments/assets/30cdd78d-461b-464f-9b43-4dda38bb8d4a" />

<img width="2452" height="1202" alt="Image" src="https://github.com/user-attachments/assets/bc44c1ce-4df2-4bf9-a395-fe97dee82521" />

<img width="2418" height="1172" alt="Image" src="https://github.com/user-attachments/assets/39b7b12f-4076-4b00-b565-0de4c2426181" />

<img width="2429" height="1249" alt="Image" src="https://github.com/user-attachments/assets/7fbbcced-e507-48d3-8249-b51c766f25e4" />

</Details>
